code/Model/Recipient/EmailRecipient.php

```diff
@@ -61,7 +61,7 @@ class EmailRecipient extends DataObject
     ];
 
     private static $has_one = [
-        'Form' => DataObject::class,
+        'Form' => UserDefinedForm::class,
```

Comment:
should this change have made it in?

Reply:
Don't think so, I thought I'd rebased it out. Although I can't see why it should be `DataObject`. Any thoughts @robbieaverill @NightJar ?

Reply:
DataObject::class makes it a polymorphic foreign key which *may* add some usefulness that I just can't think of right now :P

Reply:
Ah right. Well in that case I should switch it back most likely :)

Reply:
One example is elemental where a userform can be an element rather than a page

Reply:
The naming scheme seems to hint that it is a relation to a form as opposed to some kind of parent/host object. But without being sure, I am sure that leaving it as `DataObject` will be fine since it's a non-change.